packages/blockchain-api/src/currencyConversion/CurrencyConversionAPI.ts

```diff
@@ -52,44 +59,76 @@ export default class CurrencyConversionAPI<TContext = any> extends DataSource {
       // Same code, nothing to do
       return []
     } else if (fromCode === CGLD || toCode === CGLD) {
-      // cGLD -> X (where X !== cUSD)
```

Comment:
This works well, I've deployed it to Alfajores and have been using it successfully to implement cEUR support.

However, this code is getting quite hard to read. There are a lot of ifs and nested ifs. I wonder if there's something we can do to simplify it a little bit.

Reply:
I will refactor it